README.md

```diff
@@ -13,18 +13,19 @@ These are all built into the binary at compilation time.
 
 The ZTP manifests provided by the users are read from `./manifests`.
 The required manifests are:
-* manifests/pull-secret.yaml
-* manifests/cluster-deployment.yaml
-* manifests/agent-cluster-install.yaml
-* manifests/infraenv.yaml
+* [manifests/pull-secret.yaml](https://github.com/openshift-metal3/dev-scripts/blob/master/agent/03_agent_deployment.sh#L142-L152)
+   - Your pull secret JSON data can be [obtained from the OpenShift Console](https://console.redhat.com/openshift/install/pull-secret).
+* [manifests/cluster-deployment.yaml](https://github.com/openshift-metal3/dev-scripts/blob/master/agent/03_agent_deployment.sh#L97-L120)
+* [manifests/agent-cluster-install.yaml](https://github.com/openshift-metal3/dev-scripts/blob/master/agent/03_agent_deployment.sh#L73-L95)
+* [manifests/infraenv.yaml](https://github.com/openshift-metal3/dev-scripts/blob/master/agent/03_agent_deployment.sh#L122-L138)
+* [manifests/nmstateconfig.yaml](https://github.com/openshift-metal3/dev-scripts/blob/master/agent/03_agent_deployment.sh#L159-L194)
```

Comment:
Maybe I'm missing something but are we going to have to keep the line numbers updated when dev-scripts changes?

Reply:
I originally preferred adding a `samples` directory under `manifests` directory and updating the ReadMe with that link but there was a discussion to just point it to dev-scripts. I am in favor of adding a samples dir in this repo. WDYT? Any other options? @bfournie Thank you.

Reply:
Added samples under `./manifests/samples` dir